helmfile.d/datadog.yaml

```diff
@@ -5,7 +5,7 @@ releases:
   - name: datadog
     namespace: datadog
     chart: datadog/datadog
-    version: 2.20.4
+    version: 2.21.0
```

Comment:
https://github.com/DataDog/helm-charts/blob/main/charts/datadog/CHANGELOG.md#2210 => APM is enabled by default. Sounds good for our usage (e.g. tracing will be reported)